### PR TITLE
[FEATURE] Added documentation to add custom controller in extending b…

### PR DESCRIPTION
**Pull request by wolv-dev**
…ackend extjs module

### CODE REVIEW

File: source/developers-guide/backend-extension/index.md [hunk 1]
```diff
@@ -433,5 +437,39 @@ Ext.define('Shopware.apps.SwagExtendCustomer.view.detail.Window', {
...
+//{block name="backend/customer/application"}
+//      {$smarty.block.parent}
+//      {include file="backend/swag_extend_customer/controller/my_own_controller.js"}
+//      {include file="backend/swag_extend_customer/view/detail/my_own_tab.js"}
+//{/block}
+```
+
+After adding it to the app.js file, the fill will be loaded, but the controller will not be instanced.
+
+To instantiate your controller, you need to override an the existing controller (depends on the module you want to extend). In this file you can override existing functions of the parent controller. You can and mostly need to call the parent function with the line `me.callParent(arguments)`:
```
Comment (mitelg) [inline source/developers-guide/backend-extension/index.md]: ... override the existing...

**Comment by mitelg** [comment]
could you also please add this to the example plugin?

File: source/developers-guide/backend-extension/index.md (hunk 1 above)
Comment (uehler) [reply source/developers-guide/backend-extension/index.md]: "the fill" should be "the file", shouldn't it?

File: source/developers-guide/backend-extension/index.md [hunk 2]
```diff
@@ -433,5 +437,39 @@ Ext.define('Shopware.apps.SwagExtendCustomer.view.detail.Window', {
...
+
+```js
+//{block name="backend/customer/application"}
+//      {$smarty.block.parent}
+//      {include file="backend/swag_extend_customer/controller/my_own_controller.js"}
+//      {include file="backend/swag_extend_customer/view/detail/my_own_tab.js"}
+//{/block}
+```
+
+After adding it to the app.js file, the fill will be loaded, but the controller will not be instanced.
```
Comment (mitelg) [inline source/developers-guide/backend-extension/index.md]: the file will be loaded

File: source/developers-guide/backend-extension/index.md (hunk 1 above)
Comment (wolv-dev) [reply source/developers-guide/backend-extension/index.md]: Fixed!

**Comment by wolv-dev** [comment]
Fixed typos and added controller to example plugin

**Comment by mitelg** [comment]
thanks for your contribution @wolv-dev :+1: :tada:

**Pull request merged**